src/main/fc/cli.c

```diff
@@ -124,10 +124,20 @@ extern uint8_t __config_end;
 
 
 static serialPort_t *cliPort;
+
+#ifdef STM32F1
+#define CLI_IN_BUFFER_SIZE 128
+#define CLI_OUT_BUFFER_SIZE 64
+#else
+// Space required to set array parameters
+#define CLI_IN_BUFFER_SIZE 256
+#define CLI_OUT_BUFFER_SIZE 256
```

Comment:
There is little effect from increasing CLI_OUT_BUFFER_SIZE - Writer will flush when buffer is full and USB_CDC sends at most 32bytes per packet.